Вариант 16.3/Вариант 16.3/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+namespace Cycle
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение x");
+            int x = int.Parse(Console.ReadLine()); 
+            
+
+            Console.WriteLine("Введите значение m"); 
+            double m = double.Parse(Console.ReadLine());
+
+            Console.WriteLine("Введите значение k"); 
+            double k = double.Parse(Console.ReadLine());
```

Comment:
1. **Ошибка: неверный тип данных.** Процентная ставка по вкладу не обязательно целое число
2. Лучше писать сообщения в терминах предметного мира задачи.
3. Не используйте лишние пустые строки.
```suggestion
            Console.WriteLine("Введите процентную ставку по вкладу");
            int x = int.Parse(Console.ReadLine());             

            Console.WriteLine("Введите первоначальную сумму вклада"); 
            double m = double.Parse(Console.ReadLine());

            Console.WriteLine("Введите желаемую для накопления сумму"); 
            double k = double.Parse(Console.ReadLine());
```